docker/snapshots-creator/Dockerfile

```
@@ -29,10 +29,11 @@ COPY --from=planner /usr/src/zksync/recipe.json recipe.json
 RUN cargo chef cook --release --recipe-path recipe.json --bin snapshots_creator
 
 FROM chef AS builder
+RUN apt-get update && apt-get install -y --no-install-recommends mold && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)